OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -316,6 +316,10 @@ static boolean isInitDone() {
    static boolean isInForeground() {
       return inForeground;
    }
+   public static void setInForeground(boolean inForeground) {
```

Comment:
We don't want this to be a public method, it should only be called internally by the SDK.

Reply:
good catch! this was actually only for debugging